api/v1alpha1/search_types.go

```diff
@@ -52,13 +53,22 @@ type SearchSpec struct {
 	AvailabilityConfig AvailabilityType `json:"availabilityConfig,omitempty"`
 
 	// +optional
-	// Control list of Kubernetes resources indexed by search-collector
-	AllowDenyResources FilterSpec `json:"allowDenyResources,omitempty"`
-
-	// +optional
+	// Not part of dev preview
 	// Kubernetes secret name containing user provided db secret
 	// Secret should contain connection parameters [db_host, db_port, db_user, db_password, db_name, ca_cert]
```

Comment:
Match style of `HAHigh`.
```suggestion
	// Kubernetes secret name containing user provided db secret
	// Secret should contain connection parameters [db_host, db_port, db_user, db_password, db_name, ca_cert]
	// Not supported for development preview.
```